test/sql/inet/test_inet_escape.test

```diff
@@ -0,0 +1,223 @@
+# name: test/sql/inet/test_inet_escape.test
+# description: Test inet escape function
+# group: [inet]
+
+require inet
+
+statement ok
+PRAGMA enable_verification
+
+# escape
+query T
+SELECT * FROM (VALUES (escape('&')), (escape('&#')), (escape('&#x')), (escape('&#X')), (escape('&#y')), (escape('&#xy')), (escape('&#Xy;'))) tbl(i);
+----
+&amp;
+&amp;#
+&amp;#x
+&amp;#X
+&amp;#y
+&amp;#xy
+&amp;#Xy;
+
+query I
+SELECT escape('''<script>"&foo;"</script>''');
+----
+&#x27;&lt;script&gt;&quot;&amp;foo;&quot;&lt;/script&gt;&#x27;
+
+query I
+SELECT escape('''<script>"&foo;"</script>''', false);
```

Comment:
Can we add some tests where we run these functions over the input of a table instead of only over constants, e.g.:

```sql
CREATE TABLE escapes(esc VARCHAR, quote BOOL);
INSERT INTO escapes VALUES ('&', true), ('&', false), ('&&&&&&&&&&&&&&&&&&&&&', true), ('&&&&&&&&&&&&&&&&&&&&&', false);
SELECT escape(esc, quote) FROM escapes;
SELECT escape(esc, true) FROM escapes;
SELECT escape(esc, false) FROM escapes;
-- etc

```